.prettierignore

```diff
@@ -37,11 +37,12 @@ node_modules/
 coverage/
 
 # output from compiling Stencil projects for testing purposes
-test/**/dist/
+test/browser-compile/
+test/karma/test-sibling/
```

Comment:
I realized we were accidentally running prettier on these two output dirs. We didn't notice because CI runs Prettier in a separate task and in the past (when it wasn't a separate task), we'd run it before we generated this output.  So, this only affects local runs of Prettier. Can pull out into a different commit if desired